src/GraphQL.Tests/Subscription/SubscriptionExecutionStrategyTests.cs

```diff
@@ -608,8 +611,11 @@ private async Task<ExecutionResult> ExecuteAsync(Action<ExecutionOptions> config
         services.AddGraphQL(b => b
             .AddAutoSchema<Query>(s => s.WithSubscription<Subscription>()));
         services.AddSingleton<IObservable<string>>(Source);
-        var provider = services.BuildServiceProvider(); // not disposed intentionally
-        var executer = provider.GetService<IDocumentExecuter>();
+        var provider = services.BuildServiceProvider();
+        if (ServiceProviderDisposable != null)
+            throw new InvalidOperationException("Cannot run ExecuteAsync twice within one test");
```

Comment:
What happened here that leads to these changes?

Reply:
I can't remember; probably just general code clean-up.

The previous comment of "not disposed intentionally" really meant that it could not be disposed within this same method, which is still true (see new comment).  So now it is properly disposed at the end of the test run.

Reply:
OK. I just remember some changes with scope disposal in subscription code and I thought the same reasons may be here.